src/adapters/mermaid.ts

```diff
@@ -66,7 +66,7 @@ export class MermaidErd {
       .map((entry) => {
         const columns = entry.columns.map((column) => {
           return [
-            this.dataSource.driver.normalizeType(column),
+            this.dataSource.driver.normalizeType(column).replace(/\s+/g, "_"),
```

Comment:
@aktary can you give a little explanation for the reason for this change?

It may be very obvious e.g. spaces in column names break the diagraming tool for example.

Just give a bit of detail for the patch notes please.

Reply:
Yes, that's the reason. I assumed linking to the GH issue was sufficient. I'll add more detail in this commit.